README.md

```diff
@@ -31,6 +31,18 @@ etcd is used [in production by many companies](./ADOPTERS.md), and the developme
 
 See [etcdctl][etcdctl] for a simple command line client.
 
+## Documentation
+
+The most common API documentation you'll need can be found here:
+
+* [go.etcd.io/etcd/api/v3](https://godocs.io/go.etcd.io/etcd/api/v3)
+* [go.etcd.io/etcd/client/pkg/v3](https://godocs.io/go.etcd.io/etcd/client/pkg/v3)
+* [go.etcd.io/etcd/client/v3](https://godocs.io/go.etcd.io/etcd/client/v3)
+* [go.etcd.io/etcd/etcdctl/v3](https://godocs.io/go.etcd.io/etcd/etcdctl/v3)
+* [go.etcd.io/etcd/pkg/v3](https://godocs.io/go.etcd.io/etcd/pkg/v3)
+* [go.etcd.io/etcd/raft/v3](https://godocs.io/go.etcd.io/etcd/raft/v3)
+* [go.etcd.io/etcd/server/v3](https://godocs.io/go.etcd.io/etcd/server/v3)
+
```

Comment:
Suggestion to push this section just above `## Maintainers`. The current position breaks the old format by placing this section between the intro and the comic strip.

Reply:
Agreed - I prefer the old flow where the comic is directly after the repo initial introduction, seems a bit odd to have the comic under a `Documentation` heading.

| Old | 
|-------|
| ![image](https://github.com/user-attachments/assets/4fdf202c-9a01-4022-95ba-3af741537ea3) | 

---

| Proposed |
|---|
| ![image](https://github.com/user-attachments/assets/c77b5443-8165-4186-a024-180c9e8b4e9f) |

Worth noting we do already have a mention of documentation towards the bottom under `Next steps` perhaps we insert the new docs links there.

Reply:
Discussed during sig-etcd triage meeting - gentle ping @purpleidea are you able to address this? thanks